tests/e2e/core-tests/specs/shopper/front-end-cart-calculate-shipping.test.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests, jest/expect-expect */
+
+/**
+ * Internal dependencies
+ */
+const {
+	shopper,
+	merchant,
+	createSimpleProduct,
+	addShippingZoneAndMethod,
+	clearAndFillInput,
+	uiUnblocked,
+	selectOptionInSelect2,
+} = require( '@woocommerce/e2e-utils' );
+
+/**
+ * External dependencies
+ */
+const {
+	it,
+	describe,
+	beforeAll,
+} = require( '@jest/globals' );
+
+const config = require( 'config' );
+const firstProductPrice = config.has( 'products.simple.price' ) ? config.get( 'products.simple.price' ) : '9.99';
+const secondProductPrice = '4.99';
+const fourProductPrice = firstProductPrice * 4;
+var twoProductsPrice = eval([firstProductPrice,secondProductPrice].join('+'));
```

Comment:
I am a little hesitant to use `eval()`, even though just in tests due to the security risks associated with it. What are your thoughts on potentially casting to ints or something else here?

Reply:
I don't know what security risks you were referring to? I tried locally something but was an overcomplicated way. Could you please suggest something on how to improve it?

Reply:
> I don't know what security risks you were referring to?

Oh, sorry, I was just referring to the general recommendation not to use it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!

> tried locally something but was an overcomplicated way. 

It's very likely you tried these, but a couple of ideas I had include using the unary plus operator, so something like this:

```
var twoProductsPrice = +firstProductPrice + +secondProductPrice;
```

or

```
var twoProductsPrice = (+firstProductPrice) + (+secondProductPrice);
```

Alternatively, we could use the `parseFloat()` function:

```
var twoProductsPrice = parseFloat(firstProductPrice) + parseFloat(secondProductPrice);
```

We could even abstract the calculation out into a helper function and just call that instead so we aren't duplicating it line after line.

Let me know your thoughts, and if you think `eval()` should just be fine to use here too.

Reply:
Thanks @zhongruige !
I go with the second option, it's easy to understand and to read.

Reply:
Sounds good, thanks @veljkho!